### DISCUSSION TITLE
Welcome to TerminalImageViewer Discussions!

### DISCUSSION THREAD

**Discussion by stefanhaustein** [stefanhaustein/TerminalImageViewer#122] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sw33ber** [comment]
How do I delete terminal image viewer on linux?

**Reply by aaronliu0130** [reply]
Use your distro's package manager. Unless you built our package from source yourself, in which case delete the binary from wherever you've put it.
